### ISSUE TITLE
[Bug] Repeated cooper product name

### ISSUE THREAD

**Issue by pachamaltese** [alexandersimoes/oec#131]
Hi

I'm working with OEC's API. I have downloaded data with SITC rev 2 codes and I found that "Copper" appears twice with id 6820 and 6821.

![screenshot 2016-06-20 at 2 18 44 pm](https://cloud.githubusercontent.com/assets/10091065/16205685/45d3329a-36fb-11e6-8438-5561ad961645.jpg)

**Comment by alexandersimoes** [comment]
Good catch, SITC rev2 product 6821 should be: "Copper and copper alloys, refined or not, unwrought" which we can simplify to "Copper (unwrought)".

http://comtrade.un.org/db/mr/rfCommoditiesList.aspx?px=S2&cc=682

This will be fixed in the next release.

**Comment by pachamaltese** [comment]
thanks a lot :)

maybe more bugs will come next

shall I fix that in my R files and send it to you, maybe it helps

—————

*Mauricio Vargas Sepúlveda 帕夏*

2016-06-20 18:30 GMT-03:00 Alex Simoes <notifications@github.com>:

**Comment by pachamaltese** [comment]
here is the fixed file in R format ... it's just a re-ordering of OEC's API
but more human readable



—————

*Mauricio Vargas Sepúlveda 帕夏*

2016-06-20 18:33 GMT-03:00 Mauricio Vargas <mvargassepulveda@gmail.com>:
